server/src/main/java/org/elasticsearch/action/bulk/BulkShardRequest.java

```diff
@@ -115,7 +115,15 @@ public String toString() {
 
     @Override
     public String getDescription() {
-        return "requests[" + items.length + "], index[" + index + "]";
+        final StringBuilder stringBuilder = new StringBuilder().append("requests[").append(items.length).append("], index").append(shardId);
+        final RefreshPolicy refreshPolicy = getRefreshPolicy();
+        switch (refreshPolicy) {
```

Comment:
`If` statement might be simpler than `switch` here.

Reply:
Good point, yes, fixed in e8d591e.